Dist/FHSDK.nuspec

```diff
@@ -11,51 +11,51 @@
     <copyright>Copyright 2015</copyright>
     <tags>FeedHenry Cloud .Net Xamarin</tags>
     <dependencies>
-      
+
       <group>
-        <dependency id="Newtonsoft.Json" version="6.0.3" />
+        <dependency id="Newtonsoft.Json" version="7.0.1" />
         <dependency id="Microsoft.Net.Http" version="2.2.20" />
-        <dependency id="Unity" version="3.5.1404" />
-        <dependency id="aerogear-windows-push" version="0.0.9" />
+        <dependency id="Unity" version="4.0.1" />
+        <dependency id="aerogear-windows-push" version="0.1.0" />
       </group>
 
       <group targetFramework="net45">
-        <dependency id="Newtonsoft.Json" version="6.0.3" />
-        <dependency id="Unity" version="3.5.1404" />
-        <dependency id="aerogear-windows-push" version="0.0.9" />
+        <dependency id="Newtonsoft.Json" version="7.0.1" />
+        <dependency id="Unity" version="4.0.1" />
+        <dependency id="aerogear-windows-push" version="0.1.0" />
       </group>
 
        <group targetFramework="win8">
-        <dependency id="Newtonsoft.Json" version="6.0.3" />
-        <dependency id="Unity" version="3.5.1404" />
-        <dependency id="aerogear-windows-push" version="0.0.9" />
+        <dependency id="Newtonsoft.Json" version="7.0.1" />
+        <dependency id="Unity" version="4.0.1" />
+        <dependency id="aerogear-windows-push" version="0.1.0" />
        </group>
 
        <group targetFramework="wp8">
-        <dependency id="Newtonsoft.Json" version="6.0.3" />
-        <dependency id="Unity" version="3.5.1404" />
-         <dependency id="aerogear-windows-push" version="0.0.9" /> 
+        <dependency id="Newtonsoft.Json" version="7.0.1" />
+        <dependency id="Unity" version="4.0.1" />
+        <dependency id="aerogear-windows-push" version="0.1.0" />
        </group>
 
        <group targetFramework="wp81">
-        <dependency id="Newtonsoft.Json" version="6.0.3" />
-        <dependency id="Unity" version="3.5.1404" />
-         <dependency id="aerogear-windows-push" version="0.0.9" />
+        <dependency id="Newtonsoft.Json" version="7.0.1" />
+        <dependency id="Unity" version="4.0.1" />
+        <dependency id="aerogear-windows-push" version="0.1.0" />
        </group>
 
        <group targetFramework="wpa81">
-        <dependency id="Newtonsoft.Json" version="6.0.3" />
-        <dependency id="Unity" version="3.5.1404" />
-        <dependency id="aerogear-windows-push" version="0.0.9" />
+        <dependency id="Newtonsoft.Json" version="7.0.1" />
+        <dependency id="Unity" version="4.0.1" />
+        <dependency id="aerogear-windows-push" version="0.1.0" />
        </group>
 
        <group targetFramework="monoandroid">
-        <dependency id="Newtonsoft.Json" version="6.0.3" />
+        <dependency id="Newtonsoft.Json" version="7.0.1" />
         <dependency id="Unity" version="3.5.1404" />
```

Comment:
@edewit why do we still get some Unity 3.5.x
Would it be possible to align all to 4.0.1?